src/card.css

```diff
@@ -0,0 +1,115 @@
+.card {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  color: #060b35;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  box-sizing: border-box;
```

Comment:
Maybe better:
html {
  box-sizing: border-box;
}

Reply:
No, if I remove it from my elements, backstop tests are failing.

Reply:
i guess if you add it to html tag in reset.css it will be added to all elements